cli/src/alluxio.org/cli/cmd/info/report.go

```diff
@@ -80,9 +84,134 @@ func (c *ReportCommand) Run(args []string) error {
 		}
 		reportArg = args[0]
 	}
-	// TODO: output all in a serializable format and filter/trim as specified by flags
-	if c.format != "json" && c.format != "yaml" {
```

Comment:
since we are already trying to change the string representation of the values, i think maybe we can use the `format == "json"` to output the original json for scripts, and use `format == "readable"` to print the more human-readable format

Reply:
Is it better to use a --readable flag? I prefer to preserve --format flag for YAML format output later.

Reply:
yes. and i think it'll be better to set the human-readable output as the default option.